makefiles/libnet.mk

```diff
@@ -29,15 +30,23 @@ endif
 libnet-package: libnet-stage
 	# libnet.mk Package Structure
 	rm -rf $(BUILD_DIST)/libnet
+	mkdir -p $(BUILD_DIST)/libnet{9/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib,-dev/}
 
-	# libnet.mk Prep libnet
-	cp -a $(BUILD_STAGE)/libnet $(BUILD_DIST)
+	# libnet.mk Prep libnet-dev
+	cp -a $(BUILD_STAGE)/libnet/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) $(BUILD_DIST)/libnet-dev/
+	rm $(BUILD_DIST)/libnet-dev/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/lib/libnet.9.dylib
```

Comment:
Don't do this. Copy the required files only.

Reply:
Do you have a solution for this? CP does not seem capable of selecting all but one file.

Reply:
cp -a thing/lib/!(libnet.9.dylib)

Reply:
So I should use multiple copy statements? I just went along the find route.

Reply:
I prefer packages be intentional with what they’re putting in them as a general rule, so I copy individual files/directories rather than remove leftovers.

Reply:
`find $(BUILD_STAGE)/libnet/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)/ ! -name "libnet.9.dylib" | xargs -i cp -r -a {} $(BUILD_DIST)/libnet-dev/`

Reply:
> I prefer packages be intentional with what they’re putting in them as a general rule, so I copy individual files/directories rather than remove leftovers.

That;s understandable, can that be documented?

Reply:
@TheRealKeto ^